### PR TITLE
Conviqt tweaks

### PR DESCRIPTION
**Pull request by keskitalo**
Make the use of the conviqt TEB operator more intuitive: the beam and sky components are now synthesized automatically rather than having the user do it.

The PR also changes the way the beam file names are synthesized. Users can use any focalplane attributes in the file name and they automatically get replaced with the corresponding values in the file path.

### CODE REVIEW

**Comment by keskitalo** [comment]
@kisner @giuseppe I just merged a branch in `sotodlib` that depends on this PR. Should I expect a fairly quick turnaround on this PR or should I revert the `sotodlib` merge so that `toast_so_sim.py` workflow isn't broken?

**Comment by giuspugl** [comment]
i really like this tweaks so i would  merge this into toast , they are definitely more user friendly. thanks !

File: src/toast/ops/noise_estimation.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ class NoiseEstim(Operator):
     )
 
     pairs = List(
-        default_value=None,
+        # default_value=None,
```
Comment (tskisner) [inline src/toast/ops/noise_estimation.py:176]: Thanks, I will fix all of these in a systematic way during the fixes to the template calibration.

File: src/toast/ops/conviqt.py [hunk 2]
```diff
@@ -750,30 +755,42 @@ def _exec(self, data, detectors=None, **kwargs):
         for det in all_detectors:
             verbose = self.comm.rank == 0 and self.verbosity > 0
 
+            # Find one process that has focalplane data for this detector
+            # and broadcast the focalplane properties
```
Comment (tskisner) [inline src/toast/ops/conviqt.py:759]: In general, the same detector could have different properties from one observation to the next.  For example, a different pointing solution.  This technique is OK as long as the properties used later to select the beam file name are constant across the data set.

**Review by tskisner — APPROVED** [review]
Thanks for this work!

**Pull request merged**